plugins/mcreator-localization/help/fr_FR/potion/when_potion_applied.md

```diff
@@ -1 +1 @@
-This procedure will be executed when the potion effect is applied to an entity. 
\ No newline at end of file
+Cette procédure sera exécutée lorsque l'effet de potion sera appliqué à une entité.
```

Comment:
Tense problem. It's "Cette procédure **est** exécutée lorsque l'effet de potion **est** appliqué à une entité"